vector.yaml

```
@@ -10,6 +10,40 @@
+        .severity = upcase(string!(ansi_parsed.level))
+        .message = ansi_parsed.msg ?? .message
+      } else if exists(xml_parsed.level) {
+        # XML format detected
+        .severity = upcase(string!(xml_parsed.level))
+        .message = xml_parsed.msg ?? .message
+      } else if contains(.message, "ERROR") || contains(.message, "WARNING") || contains(.message, "CRITICAL") {
+        # Handle plain ERROR/WARNING messages
+        if starts_with(.message, "ERROR") {
+          .severity = "ERROR"
+        } else if starts_with(.message, "WARNING") {
+          .severity = "WARNING"  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)